db/migrations/0003_player_created_at_alter_player_guild.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 4.0.2 on 2024-12-28 10:27
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0002_alter_player_guild_alter_player_race_and_more'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='player',
+            name='created_at',
+            field=models.DateTimeField(auto_now=True),
+        ),
+        migrations.AlterField(
+            model_name='player',
+            name='guild',
+            field=models.ForeignKey(null=True, on_delete=django.db.models.deletion.SET_NULL, to='db.guild'),
```

Comment:
The `guild` field now allows null values and uses `SET_NULL` on deletion. This change is beneficial for maintaining data integrity by preventing orphaned records when a guild is deleted.